mysql-test/suite/galera/t/galera_multirow_rollback.test

```diff
@@ -87,3 +87,85 @@ SELECT * FROM c;
 
 DROP TABLE c;
 DROP TABLE p;
+
+
+#
+# Case 5: testing statement rollback
+#
+#SET SESSION debug_dbug="+d,wsrep_disable_fix";
+
+--connect node_1a, 127.0.0.1, root, , test, $NODE_MYPORT_1
+SET GLOBAL DEBUG_DBUG = "d,sync.wsrep_apply_cb";
+
+--connection node_1
+CREATE TABLE t1 (f1 int primary key, f2 int);
+INSERT INTO t1 VALUES (1,0);
+
+BEGIN;
+--error ER_DUP_ENTRY
+INSERT INTO t1 VALUES (2,4),(1,1);
+
+--connection node_2
+UPDATE t1 SET f2=8 WHERE f1=1;
+
+--connection node_1a
+SET DEBUG_SYNC = "now WAIT_FOR sync.wsrep_apply_cb_reached";
+
+--connection node_1
+COMMIT;
+
+--connection node_1a
+SET DEBUG_SYNC = "now SIGNAL signal.wsrep_apply_cb";
+
+--connection node_1
+SELECT * FROM t1;
+DROP TABLE t1;
+
+
+#
+# Case 6: testing statement rollback with BF abort
```

Comment:
Case 6 can also simplified and make use of wait conditions instead of debug sync points (see galera_bf_abort.test). If we make this test debug sync free it has the advantage of being run in debug and release builds.